hammer/library/config.py

```diff
@@ -66,6 +66,9 @@ def __init__(self,
         # AMI public access issue config
         self.publicAMIs = ModuleConfig(self._config, "ec2_public_ami")
 
+        # ECS image source issue config
+        self.ecs_external_image_source = ModuleConfig(self._config, "ecs_external_image_source")
```

Comment:
Maybe I miss it, but I don't see where you add `ecs_external_image_source` to our config.json conf file.

Reply:
Config changes are available in internal repo "hammer-deploy-config".
https://github.com/newscorp-ghfb/hammer-deploy-config/blob/93d86076b6e431e7a374b88ce471714a0422ea47/infprod/virginia/config/config.json#L390

Reply:
Will Hammer work without `ecs_external_image_source` in `config.json`?